lib/cmd/fh3/admin/logs/syslogs.js

```diff
@@ -0,0 +1,71 @@
+/* globals i18n */
+var common = require('../../../../common.js');
+var _ = require('underscore');
+
+module.exports = {
+  'desc': i18n._('Search for log entries associated with a request ID'),
+  'examples': [{
+    cmd: 'fhc admin syslogs --requestId="requestid12345" --projects="core,mbaas"',
+    desc: i18n._('Searching for log entries associated with request ID: requestid12345')
+  }],
+  'demand': ['requestId', 'projects'],
+  'describe': {
+    'requestId': i18n._("A request ID to search for."),
+    'projects': i18n._("A comma-separated list of project names to search for logs in."),
+    'numLogEntries': i18n._("The maximum number of log entries returned. Default is 1000."),
+    'order': i18n._("The order of entries to be displayed. [asc, desc]")
+  },
+  'perm': 'cluster/reseller:read',
+  'url': "/api/v2/logs/syslogs",
+  'method': 'post',
+  'preCmd': function (params, cb) {
+    var projects = params.projects.split(',');
+    params.order = params.order || "desc";
+
+    if(params.order !== "desc" && params.order !== "asc") {
+      return cb(new Error(i18n._("Order can either be 'asc' or 'desc']")))
+    }
+
+    //Creating indexes from the project names.
+    //See https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference-1-5.html#api-search-1-5
+    projects = _.map(projects, function (projectName) {
+      return projectName + ".*"
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)